extensions/amp-story-interactive/0.1/amp-story-interactive.css

```diff
@@ -110,3 +111,144 @@
     opacity: .4;
   }
 }
+
+/* Disclaimer icon rules */
+.i-amphtml-story-interactive-disclaimer {
+  --i-amphtml-story-interactive-disclaimer-animation-duration: 0.15s !important;
+  position: absolute !important;
+  right: 0.25em !important;
+  top: 0.25em !important;
```

Comment:
On polls without a prompt the border radius aligns:
<img width="426" alt="Screen Shot 2020-12-11 at 11 30 48 AM" src="https://user-images.githubusercontent.com/3860311/101929382-c45c7100-3ba4-11eb-9eb2-35a0dc3c018e.png">

`.5em` in the cases where the prompt is a sibling looks more consistent:
<img width="428" alt="Screen Shot 2020-12-11 at 11 35 40 AM" src="https://user-images.githubusercontent.com/3860311/101929609-1604fb80-3ba5-11eb-8ecc-624e09740688.png">

Reply:
According to mocks the icon will be outside the container so that it doesn't block the options, but this adds more complexity in the code so left it out in this PR
![image](https://user-images.githubusercontent.com/22420856/102100200-27dddd00-3df7-11eb-9fbd-7f1297ca7985.png)